internal/cmd/commands/dev/dev.go

```diff
@@ -141,7 +142,15 @@ func (c *Command) Flags() *base.FlagSets {
 		Target:  &c.flagIdSuffix,
 		Default: "1234567890",
 		EnvVar:  "BOUNDARY_DEV_ID_SUFFIX",
-		Usage:   `If set, auto-created resources will use this value for their identifier (along with their resource-specific prefix). Must be 10 alphanumeric characters. As an example, if this is set to "1234567890", the generated password auth method ID will be "ampw_1234567890", the generated TCP target ID will be "ttcp_1234567890", and so on.`,
+		Usage:   `Must be different from -secondary-id-suffix (BOUNDARY_DEV_SECONDARY_ID_SUFFIX). If set, auto-created resources will use this value for their identifier (along with their resource-specific prefix). Must be 10 alphanumeric characters. As an example, if this is set to "1234567890", the generated password auth method ID will be "ampw_1234567890", the generated TCP target ID will be "ttcp_1234567890", and so on.`,
```

Comment:
Can you put the `Must be different...` detail at the end of the usage? I think it would make it a little easier to read.

Reply:
Sure! [13c93ca](https://github.com/hashicorp/boundary/pull/2738/commits/13c93cabcaff097a4fd5f1b33303a468fe366695)